src/components/IncidentDetails/AddDefinitionOfDone/steps/DoDEvidenceScript.tsx

```diff
@@ -0,0 +1,64 @@
+import { EvidenceType } from "../../../../api/services/evidence";
+import { CodeEditor } from "../../../CodeEditor";
+
+const doDEvidenceScriptShortcutsOptions = new Map<
+  EvidenceType,
+  Record<"label" | "description" | "script", string>[]
+>([
+  [
+    EvidenceType.Topology,
+    [
+      {
+        label: "Component is Health",
+        description: "Whether the component are healthy",
+        script: "status == 'healthy'"
+      }
+    ]
+  ],
+  [EvidenceType.Check, []],
+  [EvidenceType.Config, []],
+  [EvidenceType.Log, []],
+  [EvidenceType.ConfigAnalysis, []],
+  [EvidenceType.ConfigChange, []]
+]);
+
+type Props = {
+  value: string | undefined;
+  evidenceType: EvidenceType;
+  onChange: (value?: string) => void;
+};
+
+export function DoDEvidenceScript({ value, onChange, evidenceType }: Props) {
+  const options = evidenceType
+    ? doDEvidenceScriptShortcutsOptions.get(evidenceType)
+    : undefined;
+
+  return (
+    <div className="flex flex-col space-y-4">
+      <div className="font-semibold">Script (Optional)</div>
+      <div className="flex flex-col space-y-2 h-[min(400px,calc(100vh-500px))]">
+        <CodeEditor onChange={onChange} value={value} />
+      </div>
+      {options && options.length > 0 && (
+        <>
+          <div className="font-semibold">Script Shortcuts</div>
+          <div className="flex flex-wrap w-full">
```

Comment:
Can we move this to the top, not the bottom, and use link style for the buttons?